Tribler/Core/Config/config.spec

```diff
@@ -41,6 +41,11 @@ live_edges_enabled = boolean(default=True)
 bitcoinlib_enabled = boolean(default=True)
 dummy_wallets_enabled = boolean(default=False)
 
+[chant]
+enabled = boolean(default=True)
+channel_edit = boolean(default=False)
+channels_dir = string(default='channels')
```

Comment:
I wonder whether we really need this configuration option. Maybe we could hard-code it for now? Or am I missing an important use-case for this?

Reply:
I don't understand which of three options do you point at, so let me elaborate on this:
* `enabled` is a standard option to disable the module completely (e.g. for testing purposes);
* `channel_edit` option is necessary for the dual-stack mode. We will remove it as soon as the old Channels implementation is gone;
* `channels_dir` option originally had an ability *to point beyond Tribler home dir*, but you removed that code for some reason. This was meant to provide the user an ability to move their channel directory to a place more suitable for storing rarely-used info, like torrents and downloads. Many users own an SSD and HDD, and use SSD for system apps and games, and HDD for video, music, download, torrents and other 'bulk' data. If we don't provide this option, the channel data is store twice at their primary storage device: once in the SQLite database, and once in the torrent form.

https://github.com/Tribler/tribler/issues/3897 related

Reply:
My question was about the `channels_dir` option (which is the line the comment is based on). I agree that the amount of required storage might become a problem with the channel functionality. You are also right that many users use different hard-drives/partitions for different types of content. In fact, many Tribler users have their state directory on another directory than their Tribler installation.

While this option provides flexibility, I think the ability to move the state directory to a different directory is enough for a vast majority of our user base. Also, I would like to minimize the number of configuration options. Once we have more insight into the (storage) requirements for channels, we could restore this option.

Reply:
So, you insist on removing it?

Reply:
I just discussed this with the other team members, please leave it in for now 👍

Reply:
My point is, we should either remove it completely or allow it to point beyond the Tribler config dir. Otherwise, it is useless.

Reply:
I agree. With this option, you can move it outside the Tribler config directory, right? Then just leave it in.

Reply:
Did just that